docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -712,10 +718,6 @@ public void pullImage(String imageName) {
 
             PullImageCmd pullImageCmd = this.dockerClient.pullImageCmd(image.getName());
 
-            if (this.cubeConfiguration.getDockerRegistry() != null) {
-                pullImageCmd.withRegistry(this.cubeConfiguration.getDockerRegistry());
```

Comment:
So before we were pulling with registry and now we set registry globally? Would that work for all images?